packages/simcore-sdk/src/simcore_sdk/node_ports_common/filemanager.py

```diff
@@ -411,12 +507,12 @@ async def get_file_metadata(
     :raises S3InvalidPathError
     """
     async with ClientSessionContextManager(client_session) as session:
-        log.debug("Will request metadata for s3_object=%s", s3_object)
+        _logger.debug("Will request metadata for s3_object=%s", s3_object)

```

Comment:
very confusing that you do not use the _get_file_meta_data method from here

Reply:
right, rewrote it.